### DISCUSSION TITLE
[V5] i18n / Internationalization / Translate ?

### DISCUSSION THREAD

**Discussion by Reptiluka** [adonisjs/core#2005] [Help] [answered]
As the title said, what is the best way to do this ?
I saw a package exist for V4, but not for V5.
It is planned ?
What are the best packages I can use ?

**Comment by thetutlage** [comment, marked as answer]
Yes, already part of the roadmap, https://trello.com/c/8lPTtuWl/13-support-for-translations and sorry I don't have any exact dates on when it will be released

**Reply by Reptiluka** [reply]
Thank you ! Continue your great work :)
